src/intrinsics/index.js

```diff
@@ -460,5 +460,8 @@ export function initialize(i: Intrinsics, realm: Realm): Intrinsics {
   // 8.2.2, step 12
   AddRestrictedFunctionProperties(i.FunctionPrototype, realm);
 
+  //last step: let the realm know the intrinsics are initialized
+  realm.intrinsicsInitialized = true;
```

Comment:
Looking at construct_realm.js from where this is called, it constructs a few more internal objects. So this assignment should go there at the end. And then the name `intrinsicsInitialized` is a bit off. Instead, I suggest to just reset the statistics counter, and make the increment in ObjectValue.js unconditional.